students/HarveyHerela/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+def task_one():
+    """Task 1: Print out this tuple."""
+    t1_tuple = (2, 123.4567, 10000, 12345.67)
+
+    # create a string formatter to produce:
+    # 'file_002 :   123.46, 1.00e+04, 1.23e+04'
+
+    print("file_{:0>3d} :   {:.2f}, {:.2e}, {:.3e}".format(t1_tuple[0], t1_tuple[1], t1_tuple[2], t1_tuple[3]))
```

Comment:
You can use `{:.2e}` in this case for three significant digits in the final number. For Pep8 line length, the following format is acceptable in Python:

```
    print("file_{:0>3d} :   {:.2f}, {:.2e}, {:.2e}".format(
        t1_tuple[0], t1_tuple[1], t1_tuple[2], t1_tuple[3])
    )
```